demo-shell/src/app/components/form/form.component.html

```diff
@@ -16,13 +16,14 @@ <h3>Error log:</h3>
             </div>
         </mat-tab>
         <mat-tab label="Editor">
-            <ngx-monaco-editor
-                id="adf-form-config-editor"
-                class="app-form-config-editor"
-                [options]="editorOptions"
-                [(ngModel)]="formConfig"
-                (onInit)="onInitFormEditor($event)">
-            </ngx-monaco-editor>
+            <div class="app-form-config-editor">
```

Comment:
Any reason you need to wrap ngx-monaco-editor in that extra div? Why you can't have applied class directly to ngx-monaco-editor tag?

Reply:
To increase the specificitiy. The style was not getting added by adding it directly on the editor. 
![image](https://github.com/Alfresco/alfresco-ng2-components/assets/112551534/08558d58-26b6-448f-8dab-fac6152d3065)
This is the way suggested in their doc.

Reply:
I just checked it and that seems really redundant, please see below: 
<img width="1726" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/c5e53eca-46ed-4e79-abe7-3425e966eeee">


If I apply height: 500px to ngx-monaco-editor directly then it still works like expected. You just need to write proper selector, something like: 

HTML: 
<img width="610" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/d6da6fbf-cbc1-40eb-82b3-1b207a664885">

SCSS: 
<img width="715" alt="image" src="https://github.com/Alfresco/alfresco-ng2-components/assets/115619721/21d3413e-96c8-4592-a0ff-a2e255aae333">


then it works like expected without extra div. Looks like their documentation is just wrong.

Reply:
I have made the changes that you suggested.